### DISCUSSION TITLE
problem with plots in R studio

### DISCUSSION THREAD

**Discussion by Navya156** [dat-linux/community#40] [FAQ]
I am unable to get plots through my code in R studio. I have updated to the latest version of R studio but still no use.
now, I'm getting the error: unable to establish connection with R session. And, earlier, I got the error that plots can't be displayed with this version of R studio. Please help me solve the issue. Thank you

**Comment by dat-linux** [comment]
It would help if you provided more detailed info so others can reproduce your steps.
